R/ggvenn.R

```diff
@@ -164,7 +166,7 @@ gen_label_pos_4 <- function() {
           "D",    1.5, -1.3, 0,      1)
 }
 
-prepare_venn_data <- function(data, columns = NULL, show_elements = FALSE) {
+prepare_venn_data <- function(data, columns = NULL, show_elements = FALSE, value_type = "both") {
```

Comment:
Do you think `show_percentage = TRUE` is better than `value_type = "both"`? I personally do not prefer parameters of string type, because it often makes users have to check the manual for valid values before using it.